src/main/webapp/app/exam/manage/exam-management-resolve.service.ts

```diff
@@ -23,7 +23,15 @@ export class ExamResolve implements Resolve<Exam> {
         const examId = route.params['examId'] ? route.params['examId'] : undefined;
         const withStudents = route.data['requestOptions'] ? route.data['requestOptions'].withStudents : false;
         const withExerciseGroups = route.data['requestOptions'] ? route.data['requestOptions'].withExerciseGroups : false;
-        if (courseId && examId) {
+        const isImport = route.data['requestOptions'] ? route.data['requestOptions'].forImport : false;
```

Comment:
```suggestion
        const isImport = route.data['requestOptions']?.forImport ?? false;
```
I guess that should also work